pkg/config/manifest.go

```diff
@@ -202,6 +213,55 @@ type BundleConnection struct {
 	// TODO: Need to add type once it's completed in #20
 }
 
+func UnmarshalManifest(manifestData []byte) (*Manifest, error) {
+	// Unmarshall the manifest into the normal struct
+	manifest := &Manifest{}
+	err := yaml.Unmarshal(manifestData, &manifest)
+	if err != nil {
+		return nil, errors.Wrap(err, "error unmarshaling the typed manifest")
+	}
+
+	// Identify data from the manifest that wasn't mapped, those are custom actions
+	unmappedData := make(map[string]interface{})
+	err = yaml.Unmarshal(manifestData, &unmappedData)
+	if err != nil {
+		return nil, errors.Wrap(err, "error unmarshaling the untyped manifest")
+	}
+
+	objValue := reflect.ValueOf(manifest).Elem()
```

Comment:
Can a comment be added here describing this section in layman's terms as well, like the others?

Reply:
Wondering if we should split out the untyped unmarshaling and custom action marshaling into a helper func?  Like, `DeriveCustomActions` or some such... idk; just thinking on how quick comprehension might be improved...

Reply:
I went with adding comments for now.